classes/abicheck.bbclass

```diff
@@ -26,47 +24,9 @@ python binary_audit_gather_abixml() {
         os.unlink(itempath)
 
     id = d.getVar("IMG_DIR")
-    for fn in glob.iglob(id + "/**/**", recursive = True):
-        if os.path.isfile(fn) and not os.path.islink(fn):
-                if not abicheck.is_elf(fn):
-                    continue
-
-                # If there's no error, out is the XML representation
-                ret, out, cmd = abicheck.serialize(fn)
-                bbutil.note(" ".join(cmd))
-                if not 0 == ret:
-                    bbutil.error(out)
-                    return                
-                if not out:
-                    bbutil.warn("Empty dump output for '{}'".format(fn))
-                    return
+    
+    build_all_xml(adir, id)
```

Comment:
Where is this imported?

Perhaps we also should check about this function/module name, as it doesn't not quite matching.

Thanks